src/core/server/status/get_summary_status.ts

```diff
@@ -23,21 +23,29 @@ import { ServiceStatus, ServiceStatusLevels, ServiceStatusLevel } from './types'
  * Returns a single {@link ServiceStatus} that summarizes the most severe status level from a group of statuses.
  * @param statuses
  */
-export const getSummaryStatus = (statuses: Array<[string, ServiceStatus]>): ServiceStatus => {
+export const getSummaryStatus = (
+  statuses: Array<[string, ServiceStatus]>,
+  { allAvailableSummary = `All services are available` }: { allAvailableSummary?: string } = {}
+): ServiceStatus => {
   const grouped = groupByLevel(statuses);
   const highestSeverityLevel = getHighestSeverityLevel(grouped.keys());
   const highestSeverityGroup = grouped.get(highestSeverityLevel)!;
 
   if (highestSeverityLevel === ServiceStatusLevels.available) {
     return {
       level: ServiceStatusLevels.available,
-      summary: `All services are available`,
+      summary: allAvailableSummary,
     };
   } else if (highestSeverityGroup.size === 1) {
     const [serviceName, status] = [...highestSeverityGroup.entries()][0];
     return {
       ...status,
       summary: `[${serviceName}]: ${status.summary!}`,
+      // TODO: include URL to status page
+      detail: status.detail ?? `See the status page for more information`,
+      meta: {
+        affectedServices: { [serviceName]: status },
+      },
```

Comment:
Changed the summary output for when only a single service is unavailable to be more consistent with the case where more than one is unavailable.